src/components/search/tests/SearchResults.test.jsx

```diff
@@ -207,6 +243,17 @@ describe('<SearchResults />', () => {
     expect(screen.getByText(new RegExp(errorMessageContent, 'i'))).toBeTruthy();
   });
 
+  test('renders an alert in case of an error for pathways', () => {
+    const propsForErrorPathway = { ...propsForError, contentType: CONTENT_TYPE_PATHWAY, title: PATHWAY_TITLE };
+    const errorMessageTitle = 'An error occured while finding pathways that match your search.';
+    const errorMessageContent = 'Please try again later.';
```

Comment:
nit: define these strings in a constant file so we can reuse them

Reply:
done